services/cartsService.js

```diff
@@ -0,0 +1,74 @@
+import {
+  queryCartItems,
+  queryInsertItemToCart,
+  queryUpdateItemQuantityInCart,
+  queryAddItemQuantityInCart,
+  querySubtractItemQuantityInCart,
+  queryDeleteItemInCart,
+  queryDeleteAllItemInCart,
+} from '../models/cartsDao.js'
+
+const getCartItems = async (userId) => {
+  const queryData = await queryCartItems(userId)
+  return queryData
+}
```

Comment:
```suggestion
const getCartItems = async (userId) => {
  return queryCartItems(userId)
}
```
작성해주신 코드 이외에 별다른 로직이 들어가지 않는다면 변수 선언과 비동기처리 없이 곧바로 return 해주시는것이 보다 더 깔끔한 코드가 될 수 있습니다!

다른부분들도 일괄 적용 부탁드리겠습니다~